examples/games/contributors.rs

```diff
@@ -338,10 +320,22 @@ fn contributors() -> Result<Contributors, LoadContributorsError> {
 
     let stdout = cmd.stdout.take().ok_or(LoadContributorsError::Stdout)?;
 
-    let contributors = BufReader::new(stdout)
-        .lines()
-        .map_while(|x| x.ok())
-        .collect();
+    // Take the list of commit author names and collect them into a HashMap,
+    // keeping a count of how many commits they authored.
+    let contributors = BufReader::new(stdout).lines().map_while(Result::ok).fold(
+        HashMap::new(),
+        |mut acc, word| {
+            *acc.entry(word).or_insert(0) += 1;
+            acc
+        },
+    );
 
     Ok(contributors)
 }
+
+/// Give each unique contributor name a particular hue that is stable between runs.
+fn name_to_hue(s: &str) -> f32 {
```

Comment:
Can we combine this with https://github.com/bevyengine/bevy/pull/12173 to get a nicely dispersed collection of hues?

Reply:
I played with this for funsies, but I feel like the distribution of hues from the hash alone is probably good enough.

![image](https://github.com/bevyengine/bevy/assets/200550/7e38aee4-482f-49fc-9f94-0c994851dfbc)